main/src/main/java/org/mobilitydata/gtfsvalidator/runner/ValidationRunner.java

```diff
@@ -230,7 +230,7 @@ public static void printSummary(
     }
 
     if (b.length() > 0) {
-      logger.atSevere().log(b.toString());
+      logger.atInfo().log(b.toString());
```

Comment:
Hoi @eibakke, sorry for the late response. This message is very important as it points out the fact that some notices were not verified due to previous errors like file/column formatting issues. We haven't found a good way to make it to the actual report without confusing non-technical users, as the relationship between validators and notices is unclear. I suggest setting it to `warning` if this helps to remove excessive logs on your side. Please let me know if this suggestion still is a blocker for your integration.
```suggestion
      logger.atWarning().log(b.toString());
```

Reply:
Hi @davidgamez, thank you for following up. Warning sounds like a good middle way. I'll commit the suggestion here.

Reply:
Hi David, I discussed with my colleague Vincent Paturet who pointed out that we would also prefer the ability to tune the logging from the validation runner with our logback.xml. Do you use slf4j?

Reply:
Hi @eibakke, we use FluentLogger, which uses `java.util.logging` under the hood. We do have a `logging.properties` file in the app module that can be extended to incorporate the ValidationRunner class logging levels. However, this is only for the app module. A similar solution can be implemented, depending how you are integrating the tool in your pipeline.
Reference files:
- https://github.com/MobilityData/gtfs-validator/blob/master/app/gui/src/main/resources/logging.properties
- https://github.com/MobilityData/gtfs-validator/blob/d9329f0dd4bc173cd4410bf90f2a2a909df2281b/app/gui/src/main/java/org/mobilitydata/gtfsvalidator/app/gui/Main.java#L52